### ISSUE TITLE
Yes/no design

### ISSUE THREAD

**Issue by brendt** [brendt/rfc-vote#67]
tl;dr: 

- There should be a clear distinction between yes and no arguments and counters, without solely relying on color
- There shouldn't be a negative connotation to no arguments, it are votes _for_ a no argument, not against it.

---

The concept of "yes" and "no" is pretty important in this project. However, I feel it's important to no associate any negative connotation with the "no" side, as "no" arguments are equally valid. 

For example, it doesn't make sense to style "no" as a downvote, thumbs down or something with `-`: you're actually voting _for_ a "no" argument, not against it.

<img width="198" alt="Screenshot 2023-08-13 at 07 21 37" src="https://github.com/brendt/rfc-vote/assets/6905297/b3071e4f-3ad3-4606-a1a9-1ba348458d04">

For counters, it would be nice if we have another icon or way of visualising the amount of yes and no votes, without using the downwards arrow. 

<img width="198" alt="Screenshot 2023-08-13 at 07 21 45" src="https://github.com/brendt/rfc-vote/assets/6905297/09f73ebf-c26a-4ced-8868-2e29669525cb">

For the vote button on arguments, we now distinguish solely by color. This might not be user friendly enough. I think there should be more distinction between yes and no arguments.

**Comment by brendt** [comment]
Here's one idea

<img width="486" alt="Screenshot 2023-08-13 at 07 39 21" src="https://github.com/brendt/rfc-vote/assets/6905297/f1449162-dba5-4eff-aa64-479ca5073b91">

**Comment by brendt** [comment]
Another idea I had was to work with `+1` instead of `upvote`.

**Comment by vsergiu93** [comment]
To me personally this looks good. Sure maybe some UX/UI expert could come up with something even better, but for now I would say this is quite good.

**Comment by SerhiiCho** [comment]
It totally makes sense that you are saying, I was suggesting something similar but a bit with different design:

<img width="265" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/9e1ff645-de4e-4daf-8226-3861cac24eae">

As for upvotes, I agree that we should do something like +1 or maybe a separate button like one of these:

<img width="129" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/45f6737c-0715-40fe-a7a2-9593ab700484">
<img width="138" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/6b5106d5-29a3-44d0-9edd-b75ccb4fb06c">
<img width="99" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/b20adede-23be-48e4-87b9-e32f41c6ce0b">
<img width="112" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/7651b03f-da0e-415d-ac38-63f488f02170">

Here are more ideas:
<img width="288" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/65f3d545-0040-40b0-ae0a-f1050dad4517">
<img width="292" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/a7a55427-e0fc-4de8-baad-ff5bc98f3910">
<img width="363" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/034b883e-53ba-4c6f-aede-b1d3125470ee">
<img width="253" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/b3b8300d-7e17-48e4-9b0b-b50fe36d6404">

Arrow up still makes more sense for me, as it's the most common thing that everybody is familiar with in these voting scenarios. But, since we don't have downvotes, it's a bit different situation than everybody else's.

**Comment by SerhiiCho** [comment]
To remove this negative feeling from upvoting "no" votes, we get forget about red color and replace it with dark gray:
<img width="263" alt="image" src="https://github.com/brendt/rfc-vote/assets/35465417/a5e5ac5f-28ef-4b12-9b94-1ab066c2c055">

**Comment by SerhiiCho** [comment]
Yeah, I think dark gray doesn't differ from green too much. Red color makes sense for now imo
